library/common/extensions/filters/http/platform_extension/filter.proto

```diff
@@ -0,0 +1,9 @@
+syntax = "proto3";
+
+package envoymobile.extensions.filters.http.platform_extension;
+
+import "validate/validate.proto";
+
+message Bridging {
+  string name = 1 [(validate.rules).string.min_bytes = 1];
```

Comment:
```suggestion
  string platform_api_name = 1 [(validate.rules).string.min_bytes = 1];
```

Reply:
I sort of think `name` is fine here. There's precedent, and recall that we'll be generating this. If we were to change it though, I would consider `platform_filter_name`, since not just any platform api will cut it.

Reply:
I vote for `platform_filter_name`. Usually, `name` referred to the name of the filter itself, but here it refers to the name of the _platform_ filter. So even though this is not public, I think a more specific name is good.

Reply:
https://github.com/lyft/envoy-mobile/pull/955